native/cocos/base/threading/MessageQueue.cpp

```diff
@@ -272,6 +278,9 @@ MessageQueue::~MessageQueue() {
 }
 
 void MessageQueue::consumerThreadLoop() noexcept {
+    // add tid to PerformanceHintManager
+    int32_t tid = gettid();
+    ADPFManager::getInstance().AddThreadIdToHintSession(tid);
```

Comment:
MessageQueue is used for all platforms and currently ADPF is only available on Android.

Reply:
Thanks, made the adjustment in 23d0327